src/pkg/utils/wait.go

```diff
@@ -28,6 +36,49 @@ func isJSONPathWaitType(condition string) bool {
 	return true
 }
 
+func ExecuteWaitResource(ctx context.Context, gk schema.GroupKind, namespace, name string) error {
+	cfg, err := config.GetConfig()
```

Comment:
Need to verify that flag `--kubeconfig` is picked up properly. Not the greatest solution, as passing it explicitly would be a lot nicer.

Reply:
The flag should be global in the CLI so lets ignore this for now.